modelon/impact/client/sal/service.py

```diff
@@ -402,13 +422,21 @@ def case_get_log(self, workspace_id, experiment_id, case_id):
         ).resolve()
         return self._http_client.get_text(url)
 
-    def case_result_get(self, workspace_id, experiment_id, case_id):
+    def case_result_get(
+        self, workspace_id, experiment_id, case_id, result_format='mat'
+    ):
         url = (
             self._base_uri
             / f"api/workspaces/{workspace_id}/experiments/{experiment_id}/cases/"
             f"{case_id}/result"
         ).resolve()
-        resp = self._http_client.get_octet_response(url)
+        format = ResultFormat(result_format)
```

Comment:
I think we should do this as early as possible. Consider moving this to the get_result method.